### ISSUE TITLE
Accessibility issue

### ISSUE THREAD

**Issue by MuellerMatthew** [stefanocudini/leaflet-search#63]
I have notice that your search tool is missing a default label element for the text input box which is an accessibility issue as all controls and media elements should have labels even if they are not always visible.
